specifyweb/frontend/js_src/lib/components/Interactions/InteractionDialog.tsx

```diff
@@ -288,7 +288,12 @@ export function InteractionDialog({
                 >
                   {interactionsText.addUnassociated()}
                 </Button.Info>
-              ) : model.name === 'Loan' || action.model.name === 'Loan' ? (
+              ) : (model.name === 'Loan' || action.model.name === 'Loan') &&
```

Comment:
looks hacky. can you analyze the code/use debugger to see what are the possible values of `model.name` and `action.model.name` and what are the relations between them - chances are you only need one of these 2 conditions

Reply:
(though to be fair to you, the very existence of both `model.name` and `action.model.name` in this file is a hack I wasnt' able to get rid of when refactoring this file)

Reply:
using only action.model.name is working